src/panels/CreateClusterPanel.ts

```diff
@@ -31,7 +26,8 @@ export class CreateClusterDataProvider implements PanelDataProvider<"createClust
         readonly portalUrl: string,
         readonly subscriptionId: string,
```

Comment:
As suggested in an earlier comment, maybe replace these three subscription-related parameters with `readonly subscriptionContext: ISubscriptionContext,`

Reply:
updated this. Thank you.